Parsers/ASimAuthentication/Parsers/ASimAuthenticationMicrosoftMD4IoT.yaml

```diff
@@ -15,48 +15,49 @@ References:
 Description: |
   This ASIM parser supports normalizing Microsoft Defender for IoT endpoint logs to the ASIM Authentication schema.
 ParserName: ASimAuthenticationMD4IoT
+EquivalentBuiltInParser: _ASim_Authentication_MD4IoT
```

Comment:
@KustoKing Are these changes due to positions changing, is there anything added new or got removed?
Other than EquivalentBuiltInParser property?

Reply:
Sorted columns